### PR TITLE
Move customer file endpoints to /admin/test

### PR DESCRIPTION
**Pull request by Cruikshanks**
https://trello.com/c/letq5mhW

Following on from PR #24 which removed some unused endpoints this change moves some which are only used for testing.

We've confirmed with the WRLS team they are no currently used by them. An internal review in the team also confirmed they are just used in some of our regression tests to confirm the API has carried out expected actions.

So to make a clear distinction between publically maintained and available endpoints and those just used by the SROC team this change moves the 'test only' endpoints to a new `/admin/test` path in the open api spec.

### CODE REVIEW

**Comment by Cruikshanks** [comment]
> **Cruikshanks** marked ...
> **Cruikshanks** requested ...
> **Cruikshanks** marked ...
> **Cruikshanks** requested ...

TLDR; **Cruikshanks** panicked about nothing 🤦

**Review by StuAA78 — APPROVED** [review]
